apf/apfElementOf.h

```diff
@@ -14,6 +14,8 @@
 
 namespace apf {
 
+// std::enable_if_t<std::is_standard_layout<T>::value, bool> = 0 >
```

Comment:
I assume this can be removed?

Reply:
Oh yeah my bad, that is related to type-safety for the field `valueType` access. I didn't quite get it working. I'll remove this.

Reply:
removed in a9ebda59d55e08f54ae856b2a4fe5962f5816441